profiles/common/modules/features/nexteuropa_dgt_connector/README.md

```diff
@@ -105,68 +100,65 @@ contain:
-   - Uncheck this if the site owner wants to review a translation before 
-   publishing it.
-
- - Translator plugin:
-   - This is shown for information, please do not change it.
-
- - DGT Connector plugin settings:
-   - You should see 'Main "poetry_service" variable is properly set.' if all
-   the steps above were correctly followed. Otherwise, get back and check what
-   you forgot!
-   - Counter: The counter always is *NEXT_EUROPA_COUNTER* 
-     [See DGT reference explanation]  
-   - Website identifier: This helps DGT identifying which site requested the 
-     translation.
-   - Requester code: always is *WEB* [See DGT reference explanation]
-   (#dgt-reference-number).
-   - Callback User: it must match NE-projectname where projectname is the 
-   project's code.
-   *Example: NE-ERASMUSPLUS*
-   - Callback Password: is the same as the callback_username.
-   These fields are limited to 15 characters.
-   - Poetry Username and Poetry Password.
-   On playground, you should use 'Poetry' & 'PoetryPWD'.
-   For production those credential are provided by DGT and are common to all
-   sites.
-   - Organization responsible, Author & requester: consult the values examples
-   shown below each form field as an example.
-   - Contact usernames: should be the *user names* (you connect to the network,
-  ecas or the proxy with your user name) of the persons in charge of the request.
-  This is important as only these persons can view translation details in the
-  web app.
-   - DGT contacts: The email address of your contact person at DGT. They receive
-   feedback on the translations that are commented. 
-   - Remote languages mappings: Map language codes that are not supported by the
-   DGT-Connector to the corresponding language code supported.
-   Typical example is when the site is configured for using 'Portuguese from 
-   Portugal' (code pt-pt) that should be mapped to 'pt'.
+> Configuration > Regional and Language > Translation management Translator
+
+and edits 'DGT Connector', or go to:
+
+> admin/config/regional/tmgmt_translator/manage/poetry 
+
+### TRANSLATOR SETTINGS
+- *Auto accept finished translations*: Check this if the site owner wants to review a translation before 
+publishing it.
+
+### Translator plugin:
+This is shown for information, please do not change it.
+
+### DGT CONNECTOR PLUGIN SETTINGS:
+You should see **Main "poetry_service" variable is properly set.** if all
+the steps above were correctly followed. Otherwise, get back and check what
+you forgot!
+   
+### GENERAL SETTINGS [See DGT reference explanation]
+- *Counter*: **NEXT_EUROPA_COUNTER**
+- *Requester code*: **WEB** ([more information](#dgt-reference-number))
+- *Callback User / Callback Password*: drupal credential - in lowercase, limited to 15 characters and different to poetry credential. Example: **namesiteproduction/namesiteproduction**
+- *Poetry Username / Poetry Password*: Those credential are provided by DGT. Example **NE-SITENAME/NE-SITENAME**.
```

Comment:
In the email of Vincent Paoletti, after "Regarding your instructions" there is a highlighted sentence, bigger (not red but still) that says credentials are not provided by DGT
Are you trying to kill the poor guy? :p

Reply:
I removed this information and put more information according the last communication of Vincent.